crates/build/src/lib.rs

```diff
@@ -19,28 +19,20 @@ pub async fn build(manifest_file: &Path) -> Result<()> {
     let app = toml::from_str(&manifest_text).map(BuildAppInfoAnyVersion::into_v1)?;
     let app_dir = parent_dir(manifest_file)?;
 
-    if app.components.iter().all(|c| c.build.is_none()) {
```

Comment:
I don't think we should remove this.  I'd rather have the specific "there is nothing to do" feedback than "huh it didn't do anything, what happened."